src/main/java/com/paneedah/mwc/items/guns/M79Factory.java

```diff
@@ -1,612 +1,173 @@
+                        .withFirstPersonPositioningModifyingAlt((renderContext) -> {
+                            new Transform()
+                                    .withPosition(-0.945000f, 3.625000f, -7.885000f)
+                                    .withRotation(-4.041486f, -45.595835f, -21.768277f)
+                                    .withPivotPoint(-0.350000f, -2.900000f, -0.100000f)
+                                    .withScale(3.000000f, 3.000000f, 3.000000f)
+                                    .applyTransformations();
+                        })
+
+                        .withFirstPersonHandPositioningModifying(
+                                (renderContext) -> {
+                                    new Transform()
+                                            .withPosition(2.770000f, 1.225000f, 0.140000f)
+                                            .withRotation(73.670132f, -70.659155f, 41.991085f)
+                                            .withScale(2.6F, 2.6F, 4.0F)
+                                            .withPivotPoint(0.000000f, 0.000000f, 0.000000f)
+                                            .applyTransformations();
+                                },
+                                (renderContext) -> {
+                                    new Transform()
+                                            .withPosition(-0.320000f, 0.140000f, 2.040000f)
+                                            .withBBRotation(10.0931F, 10.9576F, -10.0294F)
+                                            .withScale(3, 3, 3.5F)
+                                            .withPivotPoint(0, 0, 0)
+                                            .applyTransformations();
+                                })
+
+                        .withFirstPersonHandPositioningModifyingAlt(
+                                (renderContext) -> {
+                                    new Transform()
+                                            .withPosition(2.770000f, 1.225000f, 0.140000f)
+                                            .withRotation(73.670132f, -70.659155f, 41.991085f)
+                                            .withScale(2.6F, 2.6F, 4.0F)
+                                            .withPivotPoint(0.000000f, 0.000000f, 0.000000f)
+                                            .applyTransformations();
+                                },
+                                (renderContext) -> {
+                                    new Transform()
+                                            .withPosition(-0.320000f, 0.140000f, 2.040000f)
+                                            .withBBRotation(10.0931F, 10.9576F, -10.0294F)
+                                            .withScale(3, 3, 3.5F)
+                                            .withPivotPoint(0, 0, 0)
+                                            .applyTransformations();
+                                })
+
+                        .build())
+                .withSpawnEntityDamage(40f)
```

Comment:
No balance changes until 0.2.
```suggestion
                .withSpawnEntityDamage(70f)
```